src/evt.js

```diff
@@ -36,11 +28,29 @@ evt.prototype.raise = function() {
 };
 
 function evtInit(element) {
-
-  // handle the element/selector
-
-  return new evt();
+  if (!element || (element instanceof Element) === false) {
+    throw new Error('You must supply a DOM element or selector to evt');
+  }
+
```

Comment:
jquery accepts a selector or an element I think, but not an array. The trouble is that querySelectorAll returns a NodeList, which isn't actually an array. If we accept a collection of elements, it should bean array or a NodeList; that way the user can simply pass in the result of querySelectorAll. I reckon for now, let's keep it to single Element, or a string selector. We can add collections later if need be.